gm4_lively_lily_pads/data/gm4_lively_lily_pads/function/mechanics/right_click_detection/create.mcfunction

```diff
@@ -0,0 +1,7 @@
+# Sets up rcd
+# @s = new interaction
+# at @s
+# run from mechanics/right_click_detection/found
+
+data merge entity @s {Glowing:1b,width:0.4f,height:0.3f,response:1b,Tags:["gm4_llp_placement_rcd","smithed.entity","smithed.strict"]}
```

Comment:
Glowing has no effect in interaction entities (this issue originates from rope ladders, which should have this fixed in a separate PR).